app/controllers/session_controller.rb

```diff
@@ -75,6 +75,7 @@ def sso_provider(payload = nil, confirmed_2fa_during_login = false)
       if request.xhr?
         # for the login modal
         cookies[:sso_destination_url] = data[:sso_redirect_url]
+        render json: success_json.merge(redirect_url: data[:sso_redirect_url])
```

Comment:
This is needed because the 204 (no body) response was throwing an error on the frontend, in the `await ajax("/session/passkey/auth.json")` call.